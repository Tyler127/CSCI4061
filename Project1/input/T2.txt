aaIuytmBst6K2xi9bWXSgPHSYMfKmr1RYck7ezAlOjKHJaykdpZmnOCcr3gUSE7OmfQkxRwchd7bDVhhHxvYJWOdWEqJRzlQlX1sf3gmy5Iyc1luSrxFZDvr9KANgvNfxvIBXKO4qgUncCWdAMiqgTMeM26A4FCBtWJdWjs8HTPfmuGD06yqm3sunG5QhmoPIEeHnjDu7uVEtnMXceXfFuwFbbx7VOdg0OhWV7ksr3hWn2A5K9ZteNNqyYW6cIUXuqCEeusTJPYegpcbAUrcT8CSHkb5hs0HAwb8o9S91DOyVRtNqZFAFNfcg3dMu93bnVwLVcnzLqdHMnkyWyi5lhYS4d6FxL8gIvY2nvtaBE1le6Cugi8TcvjkXlik0Vn6YYZusdtdoxZnl4du3sMAl4H3ImIdSLKA4UPu2QDvOyjvjMFiBex938WGx4zwstNYfzR7jVIkBHc74FHtBr7GL2tzWBSufi0twx8HRV1s0BCINL23QVwHsLAAeozNul1pOAleTq0o9Ly9w8XulRtclUfsdwzZiOZVq1VFZW0PsDyJ79MGsmiAwzGTitbNcIsHw8tHGZwQqEt7r7JLmawK6j9Po6mQY0ki4sKLCCPdrwxH8M4ArEDAE8SsS3dPtWMxOEBQTogpAef49IwDil7l2bJ7rPeWK40QtW1X6R10CrhCxTAtlZElAFA7MlYZjjotXGFsPfgCpDQNI5i7q7p3f5GH4sPSWXNVBtoaMLr0It4Axoiy5PTYsEBrPdaHWQGgk0jbpiJDqBVMW411RLyCwKIAV9676hyT7ZrnhvLH4HyO5XJ1BrmHj1AGLgylO2lYA1W6Xm3r3wUHMgqZHpxNQ1P6kPOqyf9TqrhbcumxwxofplKUz8B5W1Vf5devc0u8Pg1RzQgEZN2XebmwBJc8kzMgTxNA65j1uNBYvCqv3Dq5FaFC9R4TEzw6S1ZelH9Em85xMfaAI9CF5965K133r512w6k1RokhdvD2cVnuQGBqHwM6cZ0MxNA9Y74X3gUwCXPHqkAEWM6vOHPNFjIXBcR8NW9ExjdsqGNwCJCBUjgSJZ7b93QztPkbl4WvluAMpLaJNqOfGsL1uYDpBPYK1l0RPzi8oGwRVqORCpe4XUiJSyYBIxPcUdEGAzzVBL0KHp6fUUhnZh7ZPeMTAagfEAuSEop2NX5s1YXXsZfkTNQUUDmQ1ANkkiOmG3SKgrDKlhKuHL6u4tsOy3DEO2QPKGiC3cZJ86tFx49WFBJBlPI9ILQTCtDwbxxkgLb7raftFkpUwqSzUO6uEI3Zkr7dkpq7u1o4F5SXvm0JEVFFKHJqpho8QPBxK0F8TCBRkWX7RH3r5VFAMc78lwd29nOkklYuDYqo7asO7QuoXhCRgE6TPTSuZpYRtsyxLMqbYKtoSiVCnOTINrMsurT9t4wsjqRyyWCwbYGT9Bue9qCDw98QkZoKKkC7QN1MNt3bU1MpCj2cTvy6BPvteEhW17Pe0jVj7RJ3LuBUtpZhOYebTl8F1fj0xdzdQv12Nb0wan4upPbwaxUMZwafjoV9Ic1SjGxW9sjtnJll1aMqM7PvPqiE48i3koyoTlWBv1VMd868c8HxlBTnFMdsYtYBsiXpv8btujtmfwTFNYlvOpHfKbiQY7gT5oQYhd8YxGBDHM7YpoTuDIuV3pBNFHovKmA3ZQN0iN21ZLrLrB3XZn67Ulf90bUvSNk4LznDcbEovflPVfS0Zjm8Si5shSTh6c9BNKZXHRK9E4eF0ervV3VQOKlW8s2smXSoRTVUvSoiRRQqHj33n66w6V0IwcQ0C4oSrXuEaaGJbhnPi9ZxhztKQtP5Z0PtpXseVjsFFvboHSEOioIV5NLuS5vZQMS6iey13SXEkF0eLYbfOuT9aqgHu0fvpuIrG2EeG9ZEMNcQnX3TBdBh8zgpPogK1SfcxAQSe9IaWOzDIVuXzMIHmVZ328HHC5f1mdQCpYz8MvK7938lwz4meKMioTrubZsmQ9OLnqqiyg8D0DTZE4m95hSzXPZ0V4CLOWzS2pVr4uvXn2jd42V7CZZOLO3Ht3BKErPesaRsXeFIZXtmLppHKojk4r6trEh2pKaKe0pzqPYUKFgg4Mry8BnYwpYhyxc13S25tIa5J8PJjKtXLtHAePdoEW0RS4RdY0rHGGiLYuR5QvJpCUY6mnoMKrTxsrYDwWQWzD2nzrzYqt3Zj9vXoIqIbq1JYvcIXPk07G46WFhUz10GJWQqXCrbik7LreR4DVDyajKdOoJ0QyyRz6SHMyzU66r7TXTibNLSfLbZSSk4ift0OmK1fQFDk8OBu5tJiUxEqgUMWxHsGnCaSY1YMjN08aecGR8Lss4cOichCsIFo88uIL5HcWgJh0zBoToGhBT8UELXANqy0kvLUqKtYDa1g58TEZiV5qyPCyyvIHmke52eWpcpiAsZwnTRc2GQHvvOgqpRFKCXXjVZ7UtPu3i0RrY0ACou2ceeJFEliIdFygF3gdZzWw0WZHLTHhm3ys1yd5zuIYmAOmhjpUt7Tg7Vh1URpWctZrddlNoJior6CJnHKJDKTTHehvZliK5SxL1KbnFUSlo3PadLSbx5haPXCR1WSbfOBZK2GJ7NlOohnCCnLVB1Crpp9D6IGhM7QLEQOrc2viNOWOWoiTO2l2zjNOcsdq1fxx1lKNw9nDhFKJr8wxFfLujmjKKTyujPTnfDOpoJmfEzWGnmRw3s34BEiGus6ZKOxv8bYLxSvgIap8JdVQiHAEBeaKeWIxZHvureQchqm0uYcJCykmRgzpDNG6zp4bqhDtfRTt17Q0pFNnKdcVK0oXTJeyr1Rrpcn31000cN5axrSeFzuSZhgu6dejJWHtL7JGPV4rbDgnETzzbspTl31ZpFJzjdaoScfnzIoaqRCZv0ibFm29slApobg7HInQnaBBXFcOWI0azTA9Bhsls9Jd92xerYTwuPPESmnKRVBqF5epUcukmbXuRGYSMLfyefYA17yLGfH4hR1SiWj8An200O23oWZCvWwcusvW1Gs9nbdivfewh3nmVERZFJ0NOjO0QzoGhmwCuzSmEgu2nqF4CBe8vIltf5zXKpmEUT7Jze3Pk9echJtvu1c8lC2GAjtsDMxT8dHeMaCGLoJohqwsVBSdW3UsuSLQsocWgwO6BqVONEQYelxXh3YDWViw5l31MjR8PvjanouwVcqhILtFcwZKhFfRdjeo4DJnwXwpVb6a92aSumblHZKBLnmGQrIiT5X8eMZt5hawOd2zKy5lQh6PMlxqzOLNcGh7bbbaBGk0CNr56PsaQ6POS9keBIJrdNC6FaLCz7ILtUV2LzCEeMHunZvjmXrrxp3HTzwCv0K6i06v8W2k64zAmYKHejQLU6M60ejiD6Fgr8Nef6gp6bkFElATK36P3wzfkUXBtj8YGVmAYWKKlCuFU2RboeAhigVQ9OY1xtQCYOX2JfSFgi8E96228psD5rnSKq4ONB83zNNjH1PGoWRoZ0WfIk07sCu7HeQ3wHKFhWo8mdBC8wD6aPjNWXr8I5S9ZL0CCDdqJwPj0RT9uUHSCfDQKddfGnAE6yeicffqK7aLlwntCglZsGOmX9zSOFh1jIuneIn90bcc4qwZVN39MjGXfXZNhJLYdG6GXO8SzhlFI3LWzGkLxB1CxC1GhPHPQZg9udyQ9UxkDKv9W4llpJbIzfWzDfKlOmmHA2g0650xc0DnpN9pS7WqoWxxwZHQXXQnOxISXdW67PTMsL9NLLg3WCnNsvb3xMKaWaw1XrxKd3qab8YlEEnkr4V4I5cnFMPeveTMH7gbfntT8iUxZIg9olk2gDaIC202JsXCEbmcVrdCLx3fR9JURBYEcPkkMGvRzsxy6vFUR2zrG7z9NhiAUfrncDWHm7i84TdOyUoJgkDPV6fo9gMd2Shuulh0pBttBx7mpzl9wkW7T5LzPAg5Ng6rZ07Z49Z0xUIjRxtPBlWJWlKF1NHW3VoVjKHxsuIfMRNeGnWcpfK7k8FY6rWQOLfoNRcSNDn3xfQmvmvBIN95uP7mSfivFQrX7luS4aIcCTeKrQXulTgLet6oaC833jaWafJKWoK1Oxhp7hKaz9JLpCIwaBLvcjdF7lKuo1e3tMkouENvsRCLvQYWPeJfz2W5azgD4J3k7gKQbRGanplvhmdfpRChmhzA25C8AD7Qo6RmqD4QCzRT2CR03qO44pmX6hud5w5WB38ZZaxfpcyCEtfJAWtYKUShvqlM9nRao6plgOI4NYi9Ie2iNrjphHn14z7JhRBHWNRV3Xg335aOEFuaf73ID6D5xC2VBhe0oAsFds9WXTqetDt9cAU0rCUDEjqEpOGmVjtajxa7C810nMPnfdrP4RsTxOgEbzwlFhYjHiKg8tefII5TvI85YFj6pGkuaU9WocUbyjdFOYjXnDvoxoAiKVKBJ9HohW6QRRo2hNyOeHE3z4I35dcbmv7GgKstdPrZMryKES9vmgWLRgAztifLXIpHt5qs9qPmTuivaXnEq1lTPEcQBVgzPkCCcvXLnjLLKfTsTGaSqzrSAwbpHSFa8QPoo98AdlDaV1N5bvuKvWnXo2BeRv2YpvyJSVGHve2SaaSBdZawhyeqNEmVjBoQizKj4koibbEfyljQmFxAwaBwuGxsvCpXX9JAjBevOw3gPcXVU9oBOFNTbSZFeHKSNpYzWtiz5bNwk1ENFpcBYOHOdSQS7yma4GueKxZSq9sPSymWUr1jum6GBd2YqADfSblIlPlPIgQ52mH7ElvYw3XZNWA72miagmxR3xV3lRIEo2c0Fs9LwQl5qanM4w4f366KI566ipK5RmwH7Ei0E43eJvHQOo1bAGzzzd3TZM5Xx1LK6IlkD4Yqn6f8Z6B67lCFaqzfE9FBnC36rJTIkpRNNseASrTE49RVJaxDQP5NJoTTFVg7TkomcVRs7anBWvsHA0iXiaVaFhau4aXKLrqjtwaC3DfFUnF3x5g9AFW7bFp7dmNq5NcOwaqIWZ4cbEQLfwtotOPX0LmyB2iychWc9sneQVHnHxjoSFOVaxszdwCDxqmAcwk3FOUpnDOw4IkKfXyOuuzuXMfFIBgXC5v1hBg4G6JFzdYicReI6u1rMGJ20gj7j5pwCKUEYoFidUzxIcAN4M0qVywQ4vzK4vZ0MmC5ukUUgVvg05W2KZrYQeqvZ71noBKDjNevzUHQGkT6SsI1AWO3jjIpLP6t7fmqHVrjQLTLQxEdnggfxrG4EAuTV6EuQaS0X13zOPHj84U6253OY84ScYjWRu2QfSEPIZVnLceZR57O1RjknLkMYQ7L7yqyc9zebBzJ2OSU1YDFPY5AXgwGyaaY07jGFryFgwU6jWz9q8eqakQIrfV2bY4c9TjmGw2ikqUAtBVVUj1k6LiKZFsXsRPrwEP2BILCmkEImv7Cy4N7Qlns9Xc3PYH00s8qt4iJDszW9buYECkr6Gu5jZDJaWeFADGjAVMFiQ7ldlOaILeCkBy35CepCfSipLCrA6pRSqaqDV4H2Cg7Oy7OLhKrfzsfVv3nbqSnZ9CyGUW4k1Ihl6qiQlSdb7DmCDDisc5ttlNkLTHjdE0IKx6BTdL3qIxdeJSu6QbyqTk0NuOhLCOqQuKaKnpoSjTE9fYePjiaGF7O0Dvvr7SftpkLsP7jOSkpQsRxSLh8De0nXIsr1Zjk3xy1GsHqFlsYz3wYYLiKqHJJr5eOnYSkShrsd2zh22h0IVsoBIZa3k6vd3nNmQai7P2e7Gbnt09KIloreuJY4ZFIjMYbYjRqBokm6nSI4mfNWXvXmDinAZ7WuumT9CpewCe5b5u5EWylZ4xz2nsyjst6sGb5gT7s2VkoJWJXf0RmouSVLy26crCenKTbRy15IFfTPuLgYpLIedrHxNFLUVSGHO2Xeicbq0v23ySHELUV6xK7ZHi0fm81SrIabkvAhPS2qd9WAT2MdnXPqNyQ2JFIsKcmSZTTMko0hP8t8aerVJvAUSHwfmKYyVlzkYZYsvowqrk9arsnfLGlTuOM7GY6jKbbPFAwV7jTgbX7CdGsLV7SJPFmStOgPG3FXJgWEWEN55nMHPPhdstP25bDd83gOZI6Tv27aBvm9KkVxP7SIKXWxPxYQJ6BnJxtQFD2nR0kEDFetJqWDmBVqCTaDB92GsYwmwI6iYcfaU8Clgj5EX05Baa5jNUnX5CfnxbgrHXIdOX19rr7NQh79vWrcVoUxjaqnMCw1E5UiotDhj6r5WpANA9Jwk81iaavGM82IfLZz1EDzyQAFEGqtJHED3klakxfrcMMaPJDpbJ8UJ6zxv3EBwIipEuDvs4oq0qWTRegvjZk5o9ouy5ehlMqaCNS5UHTT05rWlMMujgnWPO0ti2XhKI0PedJqFt7b8ytZRXGsmcT4ZNo2j3tMyErdxT13KPX5yxODPJGTGpkWP1wXvsRcmAaqJhjI2JApMoFkDXdv00AWAQx8OxvWenoaTfOzuI0IGF8IOFY22RR8kiSSL1kaUbWvcTcipcljzJaK2jcJ4NDwFu0ptYa8zUyiR3b1f3XyjCqIQxs80REFTfa3JReUyzzBlGT21nicPhQHqvMznXVyKLUU38a8MwMuTDNyL8Teu1OafzvqfeMUw48l06yWVWkM1CuwCjqt4lZ8Dy684DACw2ekhzEpualKrj47xqtbZKqrolzlCxXGU5NpkwrcdKeIeuASfsqGLJrcHSnI2C0s5pme8cvqioXTBbIHOjT8A5fJf9CJL07nC51SvEZwB4av233RRjI2aniujKisisCEhXqd5OnmTK5jGVTOexjz41JmUnnVsDLi5x7t58UV5rdoMbkRxwmkzs35RTINklu2I4UmTbRQLCZeTl9ai5W1tFiMisOVGXyxZaToyEHpW2bj4PTJIR6g5XZmuEdoTn3vxKc7LP5eoK71wgh81lLCOzapXxeye1kel67UcOLvDhpxkco6Rz7z8HSC4E5QFaTkMnm1wh1Lld1xI0RVG1H8Z85jz6qYdQJOB9Ead45OVu3TPaYWqJna45pxM8MwHjfOLfNQgwZo76hWjzJUaxUt6GPfU3dPDS6dPAAUVo1tZDrETRDZlVyKL6hywrwoB6Mf46DZ69yYdcEB3e3xavCR3lfQjNi3ih9wDT1jhUiP7epyTsdCk7fpnSqPC7DyhsFWsif6FIEtzSs1noCjSKkPHpjiCHSnPrS115q8xDYHR97ITaIOC5zmX2Qd5J6aXjBOPuoJcsNEy2VDGYskCWXuAZl3YuPENtrt2ug3hAH5h8m7mMcnLHGsj5xEtOKDGnMWmrHoE3p8MhntarLmIZI2PjiIBR2baglFZFvex7zOS1tAHRgboCWuCKCRIb1E4X7sv4FhzAJTEb33liZWgGe2FeM7RfH6RVwW3JPbtigvMoMlEntfZqFgLkDfwuT9RwJlRfm1YwtfJioLMxycE2l4u9vWzuH4b0oqZ2LSw34Cgns2GHElVRzNE2dqRhS8CnP0Cdp6GgbCxZpTBqC0bXCcSMyIb2imyNQerB7hrved3euXqYN4P9kintHSyOOKrKRbU3mEZAi8LoAKB2DbbhaZimVxnmkueJUBDR9y