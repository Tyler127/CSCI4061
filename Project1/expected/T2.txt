Node 0: 5b79738a639cbd7ee41e3fd596b80f9c83358c89b8a0072960f22710e7ce9836
    Node 1: 2ed1af258d63ed80a98b64c2b790aaff5be9f2e5f5a64330fe9ceb6dd2608f1b
        Node 3: 0b1c33684ed998eec06b66c928e20be704f4b4dae3e9dba64a9e2fb6b367418a
            Node 7: c214c8fefaebd1bcea276f5dffd5923fd2cba3d82825333c18350771fffc2a79
                Node 15: 81d514ded76813f6cf0c46c56a1db253d7dbbaf6133ff95382b113c6a33421cf
                Node 16: bcaa23e0a8591df5b30e0cb41d51397942ee97b7d265bd79d8f168a8b46dbbc4
            Node 8: d6fdf107047d9d26eedec584d6bd193e6bf6a428205c7a554a7638c5c31b9feb
                Node 17: 5d7283a2f2fdda80566253faa687c76a6253a5da40879b727a80eb436a6a72c5
                Node 18: 1d29c994b0f4f9e2ac1677a814889c7cef357fae97cbda49f36e66e08a2caa49
        Node 4: adaf0d3cdcbe564bc34c4d61c8d5dfd724de7970e78e19c3b5050d96cd34c60c
            Node 9: e9b332df2330c0f046686b38a30997230617fc4ca81d4a7936d9c58e21e3b0fa
                Node 19: b4ce3e6245cb0fc27c12ae87e29fc0d29b9ece2e541c68cd8411b7f9ba975dd7
                Node 20: dd381b03971a24218aa67a1375143f9c826555c089b200fcc68fc9ba22f298c2
            Node 10: 40776a57a144ce40a1143547493df9f253b5eebe71a7fc38fb29a7539dbcc1fe
                Node 21: 0f44fefdeee8a9a2f855d9bdcc86d9c3a42b5556caebeb0de76e617919b9258a
                Node 22: a6ec11fad3becb094d97d08e263cde4a3ecb8450388f44567dcba93884d440ac
    Node 2: 18a784bee3a9191b0270039980413d35e9860b2df859d3458cc66c9c85189c17
        Node 5: bfa2218db7ac131773e9ac819ec75a2944968aafc9f10b5039535c7f2f45282b
            Node 11: 5bad8ff601e8bf3e93d8296ed3455ba5318487a0436537299427a0a442ca132e
                Node 23: 8d897f9def7c9c48843fdbc59478459e916c6e98d0f2513028191789b2104612
                Node 24: 39eb2424391eecb00b94553edd7ef18c91f3ce0cd06dbc843b853a3a51969f91
            Node 12: be45c73058a345a72f300f28b4d6543b1820190ac07ef59d1eac1a0e3e5742e2
                Node 25: fdc70bb9ec88ff2bc8110f89d9959a2ea19ffcc7b16b1f0f8b7666367c688018
                Node 26: 779594332038aecbccce4b8c6cfaee9f0c7113bfd5f59a9b613982761fbe05ed
        Node 6: e502377908e6a5d82577518ad278b8901c6ffc4b3cfb61551dbf00c04097f42a
            Node 13: 7bd694b762bbc3bd314ddfd6a456f0049bed72cee506d2c4d5c039b172212a98
                Node 27: 072fdd42c4596012d23e1d44e1851bb8f76414e3c913c9becc1a30e4b38bc6b5
                Node 28: dfbd688dac3499e85420c42c64734b6d522920ea98dccddcd600000d80d5f25f
            Node 14: ee7c8933bec0724a8dcff22540fb317e4981b5fd6bf5a2bdbdfc8388c3d44150
                Node 29: 2f86246975f808263160660aedf3d70310bbaa8c2fc9d69b47fc37317747532f
                Node 30: 164ac479d9156c354087b407a312352d27cb07ab8e9c57b968c12d228901b132
